Gemfile

```diff
@@ -4,4 +4,5 @@ gemspec
 
 group :development do
   gem "mocha", "~> 0.14", :require => false
+  gem 'test-unit', '~> 3.3', :require => false
```

Comment:
Also added test-unit as a dependency, was getting a weird error when running rake:

```
$ rake
/home/kevjin/.rbenv/versions/2.6.4/bin/ruby -w -I"lib:lib:test" -I"/home/kevjin/.rbenv/versions/2.6.4/lib/ruby/gems/2.6.0/gems/rake-13.0.1/lib" "/home/kevjin/.rbenv/versions/2.6.4/lib/ruby/gems/2.6.0/gems/rake-13.0.1/lib/rake/rake_test_loader.rb" "test/test_simplecov_json.rb"

File does not exist: test/unit  
```

Reply:
I got the same error